src/componets/request/requestko.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 
 
-class GetRequestAsyncAwait extends React.Component 
-
-    
-
-{
-    constructor(props) {super(props);this.state={totalReactPackages: null} };
+class GetRequestAsyncAwait extends React.Component{
```

Comment:
Dont use Class.. Please retain to Functional Programming.

Reply:
noted po dont use class its old design